core/src/main/java/org/jruby/ext/ffi/Platform.java

```diff
@@ -84,6 +84,7 @@
         SPARCV9,
         S390X,
         ARM,
+	AARCH64,
```

Comment:
The presentation was 1 character off in _this_ view.

Then: Visiting "View", marking and dragging the cursor over and seeing that the visualization of the marked text "snapped" to a tab stop, not to individual spaces.